### ISSUE TITLE
Fails to start with pyqtgraph 0.11?

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/lasagna#246]
Showing error:
```
(lasagna) C:\Users\chris\int-brain-lab\lasagna>lasagna
Starting
Traceback (most recent call last):
  File "C:\Users\chris\Anaconda3\envs\lasagna\Scripts\lasagna-script.py", line 11, in <module>
    load_entry_point('Lasagna', 'console_scripts', 'lasagna')()
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\main.py", line 149, in run
    main(im_stack_fnames_to_load=img_stack_fnames_to_load, sparse_points_to_load=args.sparse_points,
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\main.py", line 83, in main
    tasty = Lasagna(embed_console=embed_console)
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\lasagna_object.py", line 35, in __init__
    self.setupUi(self)
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\lasagna_mainWindow.py", line 35, in setupUi
    self.graphicsView_1 = LasagnaPlotWidget(self.splitter)
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\lasagnaplotwidget.py", line 15, in __init__
    super(LasagnaPlotWidget, self).__init__(enableMenu=False,*args, **kwargs)
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\widgets\PlotWidget.py", line 54, in __init__
    self.plotItem = PlotItem(**kargs)
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\graphicsItems\PlotItem\PlotItem.py", line 141, in __init__
    self.setMenuEnabled(enableMenu, enableMenu) ## en/disable plotitem and viewbox menus
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\graphicsItems\PlotItem\PlotItem.py", line 1003, in setMenuEnabled
    self.vb.setMenuEnabled(enableViewBoxMenu)
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\graphicsItems\ViewBox\ViewBox.py", line 389, in setMenuEnabled
    self.menu.setParent(None)
AttributeError: 'NoneType' object has no attribute 'setParent'
```

Either specify pyqtgraph 0.10 or correct source of the above error.

**Comment by cskrasniak** [comment]
after switching to pyqtgraph 0.10 this error is now present: `Traceback (most recent call last):
  File "C:\Users\chris\Anaconda3\envs\lasagna\Scripts\lasagna-script.py", line 11, in <module>
    load_entry_point('Lasagna', 'console_scripts', 'lasagna')()
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pkg_resources\__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pkg_resources\__init__.py", line 2853, in load_entry_point
    return ep.load()
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pkg_resources\__init__.py", line 2444, in load
    return self.resolve()
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pkg_resources\__init__.py", line 2450, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "c:\users\chris\int-brain-lab\lasagna\lasagna\main.py", line 27, in <module>
    import pyqtgraph as pg
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\__init__.py", line 204, in <module>
    from .graphicsItems.VTickGroup import *
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\graphicsItems\VTickGroup.py", line 7, in <module>
    from .. import functions as fn
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\functions.py", line 17, in <module>
    from . import debug
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\debug.py", line 11, in <module>
    from . import ptime
  File "C:\Users\chris\Anaconda3\envs\lasagna\lib\site-packages\pyqtgraph\ptime.py", line 24, in <module>
    cstart = systime.clock()  ### Required to start the clock in windows
AttributeError: module 'time' has no attribute 'clock'`

**Comment by raacampbell** [comment]
Fixed: ff9c189e73799870398ffc1b201651fd4733e702
